pkg/api/vlabs/validate.go

```diff
@@ -27,6 +27,36 @@ var (
 		"3.1.0", "3.1.1", "3.1.2", "3.1.2", "3.1.3", "3.1.4", "3.1.5", "3.1.6", "3.1.7", "3.1.8", "3.1.9", "3.1.10",
 		"3.2.0", "3.2.1", "3.2.2", "3.2.3", "3.2.4", "3.2.5", "3.2.6", "3.2.7", "3.2.8", "3.2.9", "3.2.11", "3.2.12",
 		"3.2.13", "3.2.14", "3.2.15", "3.2.16", "3.3.0", "3.3.1"}
+	networkPluginPlusPolicyAllowed = []k8sNetworkConfig{
```

Comment:
What about the case where policy is cilium or calico and plugin is empty?

Reply:
thanks! fixed

Reply:
Still missing "" with "calico" I think

Reply:
thanks! fixed déja vu